indexing-hadoop/src/main/java/org/apache/druid/indexer/IndexGeneratorJob.java

```diff
@@ -114,13 +114,13 @@
 
     final Path descriptorInfoDir = config.makeDescriptorInfoDir();
 
-    try {
-      FileSystem fs = descriptorInfoDir.getFileSystem(conf);
-
+    try (FileSystem fs = descriptorInfoDir.getFileSystem(conf)) {
```

Comment:
Hi @Fokko, would you elaborate more on this change? Seems like `Path.getFileSystem()` returns a cached instance of a FileSystem and I guess it might be better to leave the cached one rather than removing it from the cache?